Sources/Security.php

```diff
@@ -925,6 +926,11 @@ function allowedTo($permission, $boards = null, $any = false)
 	elseif (!is_array($boards))
 		$boards = array($boards);
 
+	$cache_key = hash('md5', $user_info['id'] . '-' . implode(',', $permission) . '-' . implode(',', $boards) . '-' . $any);
+
+	if (isset($perm_cache[$cache_key]))
+			return $perm_cache[$cache_key];
```

Comment:
Indented too much.

Reply:
I don't understand what you mean.

Reply:
There are 3 tabs before this line when there should only be 2.